lib/plugins/install/install.js

```diff
@@ -30,6 +37,30 @@ class Install {
       'install:install': () => BbPromise.bind(this)
         .then(this.install),
     };
+
+    this.renameService = (name, servicePath) => new BbPromise((resolve) => {
+      const serviceFile = path.join(servicePath, 'serverless.yml');
+      const packageFile = path.join(servicePath, 'package.json');
+      BbPromise.all([
+        readFile(serviceFile, 'utf-8')
+          .then(contents =>
+            contents.replace(/service\s*:.+/gi, (match) => {
+              const fractions = match.split('#');
```

Comment:
I don't know about others, but at least we have some comments on our workshop boilerplates and other similar project templates.